.circleci/config.yml

```diff
@@ -21,13 +21,21 @@ workflows:
             tags:
               only: /.*/
 
-      - orb-tools/publish-dev:
-          orb-name: giantswarm/architect
+      - orb-tools/publish:
+          name: publish-branch
+          attach-workspace: true
           orb-path: workspace/packed/orb.yml
+          orb-ref: giantswarm/architect@dev:${CIRCLE_BRANCH}
           publish-token-variable: CIRCLECI_DEV_API_TOKEN
           requires: [orb-tools/pack]
+          filters:
```

Comment:
I get it but it's confusing on first glance. 
So the first `orb-tools/publish` is only running on branches, the second one only on tags.

Reply:
Yes, that why I added job names to better distinguish them `publish-branch` and `publish-tag`.

Reply:
This is the syntax and I doubt we can do much.